packages/shared/tailwind.config.js

```diff
@@ -127,14 +127,15 @@ module.exports = {
       1: '1',
       2: '2',
       3: '3',
-      rank: '3',
-      header: '4',
-      postNavigation: '5',
-      sidebar: '7',
-      tooltip: '8',
-      popup: '9',
-      modal: '10',
-      max: '100',
+      rank: '30',
+      header: '75',
+      postNavigation: '76',
+      sidebar: '70',
+      tooltip: '80',
```

Comment:
Tooltip should probably be higher? Like hovering a user on post modal. Or would it work ootb since it is inside the modal?

Reply:
Only thing that uses `z-tooltip` is this thing:
<img width="537" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/3c3bf9a0-84d8-4c40-aaa6-c25af362f020">

everything else is tippy or contexify which have their own z-index to 1000 or 9999.

Reply:
Ahh, nice. Thanks for the verification.